ducktape/tests/test.py

```diff
@@ -173,20 +173,74 @@ def _escape_pathname(s):
     return re.sub("^\.|\.$", "", s)
 
 
-class TestContext(Logger):
+class TestLogger(Logger):
+    def __init__(self, logger_name, log_dir, debug, max_parallel):
+        self._logger_name = logger_name
+        self.log_dir = log_dir
+        self.debug = debug
+        self.max_parallel = max_parallel
+
+    @property
+    def logger_name(self):
+        return self._logger_name
```

Comment:
When there's no computation going on, is there any reason we're not just setting it as the field name and not including these extra lines of code for wrapping that field?